spec/unit/filter_option_parser_spec.rb

```diff
@@ -90,9 +90,9 @@
         )
 
         expect(filter.to_array).to eq([
-          "/unmanaged_files/files/name=/foo",
-          "/unmanaged_files/files/name=/bar",
-          "/unmanaged_files/files/name=/file,with_comma"
+          "/unmanaged_files/name=/foo",
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.